project/HyperScalaBuild.scala

```diff
@@ -63,43 +66,129 @@ object HyperScalaBuild extends Build {
       </developers>
   )
 
+  def between(subject: String, left: String, right: String): Option[String] = {
+    val ofs = subject.indexOf(left)
+    val end = subject.indexOf(right, ofs + left.length + 1)
+    if (ofs == -1 || end == -1) None
+    else Some(subject.slice(ofs + left.length, end))
+  }
+
+  def objectName(cssTag: String): String =
+    cssTag.foldLeft("") { case (acc, cur) =>
+      if (acc.length == 0) "" + cur.toUpper
+      else if (acc.last == '-') acc.dropRight(1) + cur.toUpper
+      else acc + cur
+    }
+
+  def generateBootstrap(sourceGen: File, webJars: File): Seq[File] = {
```

Comment:
Extract all of this generation functionality out into another class. The build class is already bigger than it should be.